tests/integration/features/command/user-transfer-ownership.feature

```diff
@@ -0,0 +1,165 @@
+    And user "participant3" is participant of the following rooms (v4)
+      | id          | name         | type | participantType |
+      | moderator   | moderator    | 2    | 2               |
+      | owner       | owner        | 2    | 1               |
+
+  Scenario: Promote user if source had privileges
+    Given user "participant1" creates room "moderator" (v4)
+      | roomType | 2 |
+      | roomName | moderator |
+    And user "participant1" adds user "participant2" to room "moderator" with 200 (v4)
+    And user "participant1" promotes "participant2" in room "moderator" with 200 (v4)
+    Given user "participant2" creates room "owner" (v4)
+      | roomType | 2 |
+      | roomName | owner |
+    Given user "participant2" creates room "moderator to owner" (v4)
+      | roomType | 2 |
+      | roomName | moderator to owner |
+    And user "participant2" adds user "participant3" to room "moderator to owner" with 200 (v4)
+    And user "participant2" promotes "participant3" in room "moderator to owner" with 200 (v4)
+    Given user "participant1" creates room "from self-joined to moderator" (v4)
+      | roomType | 3 |
+      | roomName | from self-joined to moderator |
+    And user "participant1" adds user "participant2" to room "from self-joined to moderator" with 200 (v4)
+    And user "participant1" promotes "participant2" in room "from self-joined to moderator" with 200 (v4)
+    Given user "participant1" creates room "from self-joined to user" (v4)
+      | roomType | 3 |
+      | roomName | from self-joined to user |
+    And user "participant1" adds user "participant2" to room "from self-joined to user" with 200 (v4)
+    And user "participant3" is participant of the following rooms (v4)
+      | id                            | name                          | type | participantType |
+      | moderator to owner            | moderator to owner            | 2    | 2               |
+    And user "participant3" joins room "from self-joined to user" with 200 (v4)
+    And user "participant3" is participant of the following unordered rooms (v4)
+      | id                            | name                          | type | participantType |
+      | moderator to owner            | moderator to owner            | 2    | 2               |
+      | from self-joined to user      | from self-joined to user      | 3    | 5               |
+    And user "participant3" joins room "from self-joined to moderator" with 200 (v4)
+    And user "participant3" is participant of the following unordered rooms (v4)
+      | id                            | name                          | type | participantType |
+      | moderator to owner            | moderator to owner            | 2    | 2               |
+      | from self-joined to user      | from self-joined to user      | 3    | 5               |
+      | from self-joined to moderator | from self-joined to moderator | 3    | 5               |
```

Comment:
I think that have anything wrong with messages order.
https://drone.nextcloud.com/nextcloud/spreed/11041/4/4

Reply:
Fixed the unordered method... It did not replace the IDs with the numbers before sorting :D